### ISSUE TITLE
Add aria-live to dynamic elements

### ISSUE THREAD

**Issue by christianp** [numbas/editor#464]
Parts of the interface that update dynamically should have an `aria-live` attribute. 

These include:

* the content preview in the rich text editor
* warnings - particularly any JME entry boxes
* the notification and basket areas
